poly-synth/index.html

```diff
@@ -56,9 +56,11 @@
     </ul>
     <p>
     The combined output of the voices can then be modulated by bitcrushing,
-    reverb, and delay effects. If side chaining is active, the output of the
-    synthesizer will ramp towards silence when the level of the drum sample
-    exceeds the specified threshold.
+    reverb, and delay effects. If noise gate side chaining is active, then
+    the output of the synthesizer will ramp towards silence when the level of
+    the drum sample exceeds the specified threshold. The beat and its volume is
```

Comment:
It's backwards with respect to the original noise gate. However, we modify the noise gate side chain so that it behaves this way. 
`      outputChannel0[j] = (1 - weights[j]) * signalChannel0[j];
`